safenode/src/node/replication.rs

```diff
@@ -0,0 +1,155 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use crate::{
+    domain::storage::{
+        DataAddress, {ChunkStorage, RegisterStorage},
+    },
+    network::Network,
+    protocol::messages::{
+        DataRequest, DataResponse, Event, Query, QueryResponse, RegisterQuery, Request, Response,
+    },
+};
+use libp2p::PeerId;
+use std::{collections::HashSet, time::Duration};
+use tokio::spawn;
+
+pub(crate) async fn send_out_data_addrs(
+    network: Network,
+    chunk_storage: ChunkStorage,
+    _register_storage: RegisterStorage,
+) {
+    tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
don't think we need this `periodic` style check
will discuss it in detail in slack